mw-webapp/src/component/header/Header.tsx

```diff
@@ -39,12 +39,45 @@ export const Header = () => {
           onClick={user ? logOut : logIn}
           value={user ? BUTTON_LOG_OUT_VALUE : BUTTON_LOG_IN_VALUE}
         />
-        {/* //TODO: we need delete this link when after sidebar with all links to al pages will be added */}
-        <Link
-          path={pages.way.path("uuid")}
-          value={LINK_TEXT}
+        <Sidebar
+          trigger={
+            <Button value={"Navigation"} />
+          }
+          content={
+            <div className={styles.navSidebarContent}>
+              <Link
+                path={pages.allWays.path}
+                value={"All ways"}
+              />
+              <Link
+                path={pages.allUsers.path}
+                value={"All users"}
+              />
+              {user && (
+                <>
+                  <Link
+                    path={pages.way.path(user.uid)}
+                    value={"Workflow"}
+                  />
+                  <Link
+                    path={pages.user.path(user.uid)}
+                    value={"My ways"}
+                  />
+                  <Link
+                    path={pages.userProfile.path(user.uid)}
+                    value={"Profile"}
+                  />
+                </>
+              )}
+              <Link
+                path={pages.aboutProject.path}
+                value={"About"}
+              />
```

Comment:
Can we refactor it? Looks like many similar components/ Maybe better use .map

Reply:
please tell me what exactly is it better for me to map?
Create a separate array of objects in this file or something else?

example:  const navigationLinks = [
        { path: pages.always.path, value: "All ways" },
         user && { path: pages.way.path(user.uid), value: "Workflow" },
         ...]

Reply:
Yea,  it will be better

Reply:
OK, thanks) I did it)